README.md

```diff
@@ -5,28 +5,28 @@
 
 Mudlet is a quality MUD client, designed to take mudding to a new level.
 
-It’s a new breed of a client on the MUD scene – with an intuitive user interface, a specially designed scripting framework, and a very fast text display. That, along with cross-platform capability and an open-source development model results in a very likable MUD client.
+It’s a new breed of a client on the MUD scene – with an intuitive user interface, a specially designed scripting framework, and a very fast text display. That, along with cross-platform capability and an open-source development model result in a very likable MUD client.
 
 ## Easy to use client
 
-We’re big on usability, and as such, creating an easy to use client and interface is one of the defining goals of the project. This applies to both the power users and plain gamers – everyone will feel at home with Mudlet, without having to waste too much time figuring out how to do something. Comments on display improvement are always welcome, and we’ll continue refining the design to perfection continually.
+We’re big on usability, and as such, creating an easy to use client and interface is one of the defining goals of the project. This applies to both the power users and plain gamers – everyone will feel at home with Mudlet, without having to waste too much time figuring out how to do something.
```

Comment:
Apart from setting up a mapping system - but hey, lets not put the punter's off. 😈

Reply:
I have yet to attempt to do anything with the mapper. :hide: